### PR TITLE
Clean gemfile and upgrade to 0.26.5

### PR DESCRIPTION
**Pull request by paulinebessoles**
#### :tophat: Description
Make gems more visibles so we can see clearly which Decidim and Decidim external modules are deployed. 

#### Tasks
- [ ] Add specs
- [ ] Add note about overrides in OVERLOADS.md

### :camera: Screenshots
*Please add screenshots of the changes you're proposing if related to the UI*

### CODE REVIEW

**Review by armandfardeau — DISMISSED** [review]

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
As discussed, PR aims to bump to Decidim v0.26.5 however version doesn't change in Gemfile.lock

**Comment by paulinebessoles** [comment]
I still have some work to do on this PR, some things seems to have fail!

**Comment by Quentinchampenois** [comment]
> I still have some work to do on this PR, some things seems to have fail!

Please pull before or rebase this branch because I just merged conflict in Gemfile.lock !

**Review by armandfardeau — DISMISSED** [review]

File: Gemfile [hunk 1]
```diff
@@ -17,10 +17,10 @@ gem "decidim-cache_cleaner"
 gem "decidim-decidim_awesome"
 gem "decidim-friendly_signup", git: "https://github.com/OpenSourcePolitics/decidim-module-friendly_signup.git"
 gem "decidim-homepage_interactive_map", git: "https://github.com/OpenSourcePolitics/decidim-module-homepage_interactive_map.git", branch: DECIDIM_BRANCH
-gem "decidim-ludens", git: "https://github.com/OpenSourcePolitics/decidim-ludens.git"
+gem "decidim-ludens", git: "https://github.com/OpenSourcePolitics/decidim-ludens.git", branch: "release/0.26-stable"
```
Comment (eliegaboriau) [inline Gemfile:20]:
```suggestion
gem "decidim-ludens", git: "https://github.com/OpenSourcePolitics/decidim-ludens.git", branch: DECIDIM_BRANCH
```

**Review by eliegaboriau — APPROVED** [review]
LGTM, you should may use the DECIDIM BRANCH for ludens too

**Review by eliegaboriau — APPROVED** [review]

**Review by Quentinchampenois — APPROVED** [review]
LGTM  !

**Pull request merged**